guides/front_end/vue_quick_start.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Vue3 quick start — Meilisearch documentation
+description: Integrate a search-as-you-type experience into your Vue app.
+---
+
+# Vue3 quick start
+
+## 1. Create a Vue application
+
```

Comment:
Perhaps a quick, one-sentence description of what `npm create` will do? 

e.g. "Run the `npm create` tool to install base deps and create your app folder structure"